homeassistant/components/template/weather.py

```diff
@@ -130,6 +145,11 @@ class WeatherTemplate(TemplateEntity, WeatherEntity):
     """Representation of a weather condition."""
 
     _attr_should_poll = False
+    _attr_supported_features = (
+        WeatherEntityFeature.FORECAST_DAILY
+        | WeatherEntityFeature.FORECAST_HOURLY
+        | WeatherEntityFeature.FORECAST_TWICE_DAILY
+    )
```

Comment:
This needs to be set in `__init__` based on what forecast templates are provided